backend/app/video_extractor.py

```diff
@@ -0,0 +1,153 @@
+import os
+import subprocess
+import boto3
+from celery import Celery
+from botocore.exceptions import NoCredentialsError
+from config import (
+    AWS_ACCESS_KEY, AWS_SECRET_KEY, S3_BUCKET_NAME, AWS_REGION,
+    CELERY_BROKER_URL, CELERY_BACKEND_URL,
+    SOFT_TIME_LIMIT, TIME_LIMIT, MAX_CONVERT_TRIES
+)
+import shutil
+
+# Setup Celery
+celery = Celery(
+    "video_extractor",
+    broker=CELERY_BROKER_URL,
+    backend=CELERY_BACKEND_URL,
+
+)
+
+# AWS S3 Client
+s3_client = boto3.client(
+    "s3",
+    aws_access_key_id=AWS_ACCESS_KEY,
+    aws_secret_access_key=AWS_SECRET_KEY,
+    region_name=AWS_REGION,
+)
+
+
+def convert_pptx_with_unzip(pptx_file_path, output_dir):
+    """
+    Convert the .pptx file to a raw directory structure using Unzip.
+
+    Args:
+        pptx_file_path (str): Path to the pptx file.
+        output_dir (str): Path to the output directory (*_output)
+    """
+    os.makedirs(output_dir, exist_ok=True)
+    unzip_command = [
+        "unzip",
+        "-j",
+        pptx_file_path,
+        "ppt/media/*",
+        "-d",
+        output_dir,
+    ]
```

Comment:
```suggestion
    unzip_command = ["unzip", "-j", pptx_file_path, "ppt/media/*",  "-d", output_dir]
```

I think I'd prefer this for the conciseness, thoughts?